src/services/identity/DeploymentClient.ts

```diff
@@ -88,6 +96,28 @@ class DeploymentClient {
       JEKYLL_ENV: branchName === "master" ? "production" : "staging",
     },
   })
+
+  generateUpdatePasswordInput = (
```

Comment:
nit: could we separate out the concerns of with two functions of
`generateUpdatePasswordInput`
and 
`generateDeletePasswordInput`?

Reply:
Sure, changed in https://github.com/isomerpages/isomercms-backend/pull/806/commits/592ab87a469d24dd1564872dd7d79bfba4ee5c6b